ch04.adoc

```diff
@@ -265,11 +273,8 @@ The time zone offset _Z_ must be in one of the following five formats, where num
 
 ** three digits, of which the first is the hour (0--9) e.g. **`530`**.
 
-If the time zone offset is the letter `Z` or begins with a sign, the space before it may be omitted.
-
-While the default (of omitting the _Z_ component) is an offset of zero, we suggest that a zero offset be specified to avoid any confusion where omitting it might be misunderstood as indicating local time.
-
 For example, **`seconds since 1992-10-8 15:15:42.5 -6:00`** indicates seconds since October 8th, 1992 at 3 hours, 15 minutes and 42.5 seconds in the afternoon, in a time zone where the datetime is six hours behind the default.
+To translate a given time coordinate to the time zone of the reference datetime, the time zone offset should be added to the time coordinate before converting it to a datetime.
```

Comment:
I don't understand this ... a time coordinate is, by definition, an elapsed time since the reference datetime, so what are we converting it to?

Reply:
I don't know what it was supposed to mean. I have deleted it.